dataset.py

```diff
@@ -0,0 +1,189 @@
+import datasets
+from datasets import Dataset, Audio, Value
```

Comment:
repeated imports,
we can remove `import datasets` since it is already covered by `from datasets import Dataset, Audio, Value`

Reply:
Indeed that's a bit messy. There were some other things being used from datasets, so I changed it to just import the module now and refer to datasets.Dataset, datasets.load_dataset() etc.